### DISCUSSION TITLE
How to make MP4 play directly in the browser?

### DISCUSSION THREAD

**Discussion by jackadam1981** [requarks/wiki#6657] [Help / Questions]
How to make MP4 play directly in the browser?

I am continuing to import some of my knowledge base information, through git, html files, it is really convenient, and I have also imported some pictures in gif format, using the following example:
`<a href="001.gif" target="_blank">001GIF</a>`
The effect is very good, a new window will appear after clicking, and the GIF animation will be played.
When I added the MP4 format, there was a problem. The file in the mp4 format became a download and could not be played, even if I canceled the setting of the new window.

Because I uploaded html with git, I tested it locally, and the browser click can play directly, which proves that the browser settings are correct.

Maybe there is a bug, insert media, I don’t know what it supports, directly fill in the relative path of the mp4 file, fill in the address that can be downloaded, and the address that other servers can play directly, all prompt me this meida URL is not supported.

Of course, I deployed it using docker. Is it a problem with the WEB server in docker?

**Comment by jackadam1981** [comment]
I believe that many people will need a knowledge base to manage some video materials.
Some tutorials are indeed in video format. Modern browsers also support direct browser playback without the need for a streaming server. I have used apache2.2 to play mp4 directly. I will test it with nginx.

**Comment by jackadam1981** [comment]
Nginx play MP4 in browsers fine.

**Comment by NGPixel** [comment]
You need to disable "Force Download of Unsafe Extensions" under Admin > Security.

**Comment by jackadam1981** [comment]
Thanks,this is exactly what i need.
